doc/api/readline.md

```diff
@@ -447,6 +447,10 @@ autocompletion is disabled when copy-pasted input is detected.
 
 If the `stream` is a [TTY][], then it must be in raw mode.
 
+*Note* This is automatically called by any readline instance on its `input`,
```

Comment:
A few small nitpicks:

Nit: Put a `:` after `*Note*` to match the other five instances in the doc: `*Note*: This is automatically...`

Nit: Remove the comma after `input`.